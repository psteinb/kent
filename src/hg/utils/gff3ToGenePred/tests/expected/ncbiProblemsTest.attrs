rna33064	ID	rna33064
rna33064	gbkey	tRNA
rna33064	product	tRNA-Pro
rna33063	ID	rna33063
rna33063	gbkey	tRNA
rna33063	product	tRNA-Thr
gene25009	ID	gene25009
gene25009	Dbxref	GeneID:807931
gene25009	Name	CYTB
gene25009	gbkey	Gene
gene25009	gene	CYTB
gene25009	gene_synonym	cytB
