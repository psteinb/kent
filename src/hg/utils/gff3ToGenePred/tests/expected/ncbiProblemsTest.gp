rna33064	NC_001700.1	-	16247	16314	16314	16314	1	16247,	16314,	0	rna33064	none	none	-1,
rna33063	NC_001700.1	+	16177	16247	16247	16247	1	16177,	16247,	0	rna33063	none	none	-1,
