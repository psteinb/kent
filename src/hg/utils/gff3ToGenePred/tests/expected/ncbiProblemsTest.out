Error: genePred from GFF3: input/ncbiProblems.gff3:22: gene14757 exon 2 (NW_001956548.1:26533059-26533133) overlaps previous exon (NW_001956548.1:26532888-26533060)
Warning: skipping: invalid genePred created: gene14757 NW_001956548.1:26531106-26534144
Warning: skipping: invalid strand for mRNA cds29481
2 warnings converting GFF3 file: input/ncbiProblems.gff3
